README.md

```diff
@@ -6,7 +6,11 @@ The ESLint generator for [Yeoman](https://yeoman.io/). This generator is intende
 
 First and foremost, you must have [Node.js](https://nodejs.org/) and npm installed. If you don't have Node.js installed, please download and install the latest version.
 
-You must also install Yeoman, if you don't have it installed already. To install Yeoman, you can run this command:
+> **Requirements**
+>
+> - Node.js v14.17.x, v16.x and >=18.0.0
```

Comment:
Can we copy the versions straight from package.json? Might as well just match it exactly, which will also make it easier to keep it in sync when we update those versions in package.json in the future.

Reply:
Done